### DISCUSSION TITLE
Score summary button changes?

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29735] [Q&A] [answered]
I downloaded the newest lazer, and i was (nicely) surprised by the fact that you can now favorite and add the beatmap the score was set on to a collection.

![image](https://github.com/user-attachments/assets/e4d3211a-64fa-4e23-9be1-51b5279a42d0)
This is how the current buttons look.
While it makes sense for Download/Watch replay and retry to be highlighted, i feel like they are a little to wide.
They didn't feel as big before, because there only were 2 buttons, but now they have a comparison right next to them, and also now they aren't on the center, which also makes it feel weird _In My Opinion_. Not asking for them too be the same size, but just chop their size a little if possible.

Since you are now adding new buttons to the summary screen, how about adding an "export replay" button? would be pretty nice _In My Opinion_.
Also a possibility to export replays as .mp4 or as videos overall would also be very nice _In My Opinion_, since most of my friends don't play osu! but i just like bothering them with my replays, and using OBS for that isn't neceseraily a problem, but it would definitely be faster (This also has other uses, but i listed my favorite one).

Let me know how do you feel about these changes.

**Comment by Joehuu** [comment, marked as answer]
> how about adding an "export replay" button

Yeah, maybe. It's hidden behind a keybind (F2) and context menu when right clicking local score in song select right now.

> export replays as .mp4 or as videos overall

There is https://github.com/ppy/osu/discussions/12986. I've unlocked it (because it was heated before) so you can secretly upvote, but don't comment unless you are a developer and implemented it.
